yt/chyt/controller/internal/jupyt/configs.go

```diff
@@ -0,0 +1,46 @@
+package jupyt
+
+import (
+	"context"
+	"encoding/json"
+	"go.ytsaurus.tech/yt/go/ypath"
+	"go.ytsaurus.tech/yt/go/yt"
+)
+
+type YTServerConfig struct {
+	YTProxy          string `json:"yt_proxy"`
+	YTAuthCookieName string `json:"yt_auth_cookie_name"`
+	YTAcoName        string `json:"yt_aco_name"`
```

Comment:
Golang [codestyle](https://google.github.io/styleguide/go/decisions#initialisms) dictates abbreviations to be the same case
```
YTACOName
YTACONamespace
YTACORootPath
```

Also names `ACOName`, `ACONamespace`, etc are clear to me without `YT` prefix, so probably we can get rid of it. But both cases are OK to me, so choose whichever you like the most

Reply:
I think it would be be better to specify that this value is related to YT.